instant.ts

```diff
@@ -149,21 +191,36 @@ const setEnvVars = (packageInfo: PackageInfo) => {
 
     envVars.push({
       'Environment Variable': envVar,
-      'Default Value': defaultEnv,
-      'Updated Value': env[envVar]
+      'Current Value': env[envVar]
     })
   }
 
   if (envVars?.length > 0) {
-    console.table(envVars)
+    console.log(
+      `🛠️ Config set for ${packageInfo.metadata.name} (${packageInfo.metadata.id}):`
+    )
+    console.table(
+      envVars.map(
+        ({ 'Environment Variable': envVar, 'Current Value': currVal }) => ({
+          'Environment Variable':
+            envVar.length > 50
+              ? `${envVar.substring(0, 50)}...[trunc]`
+              : envVar,
+          'Current Value':
+            currVal && currVal.length > 50
+              ? `${currVal.substring(0, 50)}...[trunc]`
+              : currVal
```

Comment:
How about a small helper to avoid code redundancy ?

Reply:
Cool good idea